test/controllers/TransferAgreementControllerSpec.scala

```diff
@@ -216,6 +210,14 @@ class TransferAgreementControllerSpec extends FrontEndTestHelper {
     )
   }
 
+  private def instantiateTransferAgreementController(securityComponents: SecurityComponents) = {
+    val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+    val transferAgreementService = new TransferAgreementService(graphQLConfiguration)
+
+    new TransferAgreementController(securityComponents, new GraphQLConfiguration(app.configuration),
+      transferAgreementService, getValidKeycloakConfiguration, langs)
+  }
+
   private def stubTransferAgreementResponse(transferAgreement: Option[ata.AddTransferAgreement] = None, errors: List[GraphQLClient.Error] = Nil): Unit = {
     val client = new GraphQLConfiguration(app.configuration).getClient[ata.Data, ata.Variables]()
 
```

Comment:
Might be worth putting in a comment about Intellij showing an error at line 224, when in fact there isn't one. Just to avoid any confusion in the future.